### ISSUE TITLE
BERT model doesn't learn

### ISSUE THREAD

**Issue by nmb-paperspace** [zzweig/huggingface-graphcore#2]
The BERT-Large model in `Introduction/introduction_to_optimum_graphcore.ipynb` does not learn: the loss outputs are NaN throughout.

Cell code is

`trainer.train(resume_from_checkpoint=False)`

resulting in output

`{'loss': nan, 'learning_rate': 0.0001927536231884058, 'epoch': 0.07}`

etc., to the end of training.

The corresponding exact match and F1 scores are then not useful and the model does not answer the text comprehension question.

**Comment by nmb-paperspace** [comment]
Original repo https://github.com/huggingface/optimum-graphcore rather than this one model isn't learning either

**Comment by internetoftim** [comment]
Need `transformers==4.20.0`, the pip install defaults to 4.21

**Comment by nmb-paperspace** [comment]
Adding this resolves the issue
